editor/src/components/filebrowser/fileitem.tsx

```diff
@@ -489,11 +478,13 @@ class FileBrowserItemInner extends React.PureComponent<
     }
   }
 
-  onItemMouseDown = () => {
-    if (this.props.fileType !== 'ASSET_FILE' && this.props.fileType !== 'IMAGE_FILE') {
-      this.props.setSelected(this.props)
-      if (this.props.fileType != null && this.props.fileType !== 'DIRECTORY') {
-        this.props.dispatch([EditorActions.openCodeEditorFile(this.props.path)], 'everyone')
+  onItemMouseDown = (e: React.MouseEvent) => {
+    if (e.button < 2) {
```

Comment:
I thought so ("everything but the right mouse button"), but I don't actually _own_ an input device with a middle mouse button. Think it should just be `0` here?

Reply:
I think `0` would make the most sense. The middle mouse button tends to be for either a different result when selecting (e.g. when selecting a link it will open a new tab) or nothing. I can't think of a case off the top of my head where it behaves the same as the left mouse button.

Reply:
Open new tab?? Now I want a mouse with a middle button :) Fixed.